src/main/java/signatures/Version1.java

```diff
@@ -0,0 +1,67 @@
+package signatures;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+public class Version1 {
```

Comment:
Any reason not to just call it "Signature"?  Are we expecting more than one version of how this works.

Reply:
I don't know, I did originally call it `Signature` but it resulted in there being a method like this:
```
Signature signature = new Signature("test-role")
String sigString = signature.getSignature()
```
I thought that looked really weird, and I also thought versioning was possible because AWS is currently on version 4, I think, of their signing string, so I thought maybe it would read better if I changed it. I'm not sure it's any less awkward though.

Do you think I should call the object a `Signature`? Or something else?

Reply:
I think Signature is fine for the object, unless maybe PCFSignature perhaps?

Reply:
Ah yes! I think I will stick with Signature, then.